src/Features/Navigation/CoreMenu.php

```diff
@@ -42,6 +43,11 @@ public function init() {
 	 * Add registered admin settings as menu items.
 	 */
 	public static function get_setting_items() {
+		// Let the Settings feature add pages to the navigation if enabled.
+		if ( Loader::is_feature_enabled( 'settings' ) ) {
```

Comment:
Should this also include this condition: `&& 'yes' === get_option( 'woocommerce_settings_enabled', 'no' )` ?
Because if I enable the new nav, it only shows the `Analytics` setting for me, but maybe that's intended.

Reply:
> Should this also include this condition && 'yes' === get_option( 'woocommerce_settings_enabled', 'no' ) 

Good call.  Used the new `Features:is_enabled( 'settings' )` for this.

>If I enable the new nav, it only shows the Analytics setting for me, but maybe that's intended.

I think that may be related to this bug - https://github.com/woocommerce/woocommerce-admin/issues/6180

Reply:
Thanks for making that change, and indeed this was also fixed with #6180